pca-ssm/cfn/ssm.template

```diff
@@ -92,7 +92,7 @@ Parameters:
       a JSON Object with key/value pairs, where the LLM will run one inference on each key/value pair with the value
       containing the prompt. Use {transcript} as a placeholder for where the call transcript will be injected.
     Default: >-
-      Human: Answer all the questions below as a json object with key value pairs, the key is provided, and answer as the value, based on the transcript. Only return json.
+      Human: Answer all the questions below as a json object with key value pairs, based on the transcript. Use the text before the colon as the key. Only return json. Use gender neutral pronouns. Skip the preamble; go straight into the json.
```

Comment:
Same comment as above.. Missing a <br><br> here to match the Anthropic prompt docs `\n\nHuman:`